dashboard/app/models/skill.rb

```diff
@@ -8,9 +8,42 @@
 #  concept             :string(255)
 #  created_at          :datetime         not null
 #  updated_at          :datetime         not null
+#  key                 :string(255)      not null
+#
+# Indexes
+#
+#  index_skills_on_key  (key) UNIQUE
 #
 class Skill < ApplicationRecord
   validates :description, presence: true
 
   has_and_belongs_to_many :levels, join_table: 'levels_skills'
+
+  def seeding_key(seed_context)
+    {'skill.key': key}.stringify_keys
+  end
+
+  def self.setup
```

Comment:
Eventually, levelbuilders will create Skills, but this seemed like a good intermediary to get things working. They are the same skills that we currently hardcode [here](https://github.com/code-dot-org/code-dot-org/blob/3aed0d963bc64d48dbfb4560d4fadbe122f6b9c4/dashboard/app/helpers/ai_system_prompts/evaluate_system_prompt_helper.rb#L25).

Reply:
The skills listed here should include a human-readable key. The key should be used in evaluate_system_prompt_helper.rb to identify the key, rather than duplicating its contents.